pyupgrade/_plugins/timeouterror_aliases.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+import ast
+from typing import Iterable
+
+from tokenize_rt import Offset
+
+from pyupgrade._data import register
+from pyupgrade._data import State
+from pyupgrade._data import TokenFunc
+from pyupgrade._plugins import exceptions
+
+EXC_REWRITE_TARGETS = (
+    exceptions.ExceptionRewriteTarget('socket', 'timeout', (3, 10)),
+    exceptions.ExceptionRewriteTarget('asyncio', 'TimeoutError', (3, 11)),
+)
```

Comment:
oh I meant that these can all just live in the `exceptions` plugin -- no need to separate these